src/components/AddItem.js

```diff
@@ -5,7 +5,7 @@ import { useHistory } from 'react-router-dom';
 const AddItem = () => {
   const [item, setItem] = useState('');
   const [frequency, setFrequency] = useState(0);
-  const [lastPurchasedDate, setLastPurchasedDate] = useState(null);
+  const [lastPurchasedDate] = useState(null);
```

Comment:
Won't this always just be a `null` value? If so, would it make sense to remove this state completely?

Reply:
You're right! I'm going to remove it, adjust the item template to just use lastPurchasedDate: null and re-commit before merging.